docs/dev/api/0.1.1/objects.yaml

```diff
@@ -6,14 +32,17 @@ Config:
       type: string
       example: /Recipes
       description: The folder in the user's files that contains the recipes
+      nullable: true
```

Comment:
maybe we should make the parameters not required instead of nullable?

I would need to look in the code to see if the server accepts a json with only one attribute but setting it to null is kinda wired isn't it?

Reply:
I can tell you that it is in fact optional. I will create a small PR to fix that. Honestly, I was more concerned to get the changes separated for the upcoming 0.1.1 API vesion.